.github/CODEOWNERS

```
@@ -1,7 +1,2 @@
 # Global
-*                   @alphagov/di-sse-approvers          @alphagov/di-sse-frontend
-*                   @govuk-one-login/sse-approvers      @govuk-one-login/sse-frontend
-
-# Content
-/src/views/			@alphagov/di-sse-content			@alphagov/di-sse-approvers
-/src/views/			@govuk-one-login/sse-content	    @govuk-one-login/sse-approvers
+*   @govuk-one-login/orchestration-team
```

```suggestion
*   @govuk-one-login/orchestration-team @govuk-one-login/sse-approvers @govuk-one-login/sse-sres 
```
I don't want to block these on approvals from the old Orch team